### ISSUE TITLE
Using `kops get clusters -o json` Should Return Valid JSON

### ISSUE THREAD

**Issue by brandoncole** [kubernetes/kops#2289]
When using the command:

```bash
kops get clusters -o json
```

The resulting output is not valid JSON.  The format is:

```
{...}{...}{...}
```

vs:

```
[{...},{...},{...}]
```

If this is intentional I'd be curious to know the reasons why, otherwise I'd be happy to help submit a fix to https://github.com/kubernetes/kops/blob/master/cmd/kops/get_cluster.go#L153

**Comment by chrislovecnm** [comment]
No reason I can see ... needs to be valid json

**Comment by Steniaz** [comment]
It is not only get cluster, kops get instancegroups -o yaml also returns wrongly formatted json

**Comment by chrislovecnm** [comment]
Can someone define a schema for me?

**Comment by stijndehaes** [comment]
What doe you exactly mean by a schema?
The format @brandoncole mentioned should fix everything for my use case.

```[{...},{...},{...}]```

**Comment by chrislovecnm** [comment]
Closing, fix was merged today

**Comment by chrislovecnm** [comment]
Will be released on 1.8
